test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1416,6 +1416,77 @@ def forward(self, x, scale):
         vec_amx = VecAMX()
         self._check_amx_counter(vec_amx)
 
+    @inductor_config.patch({"freezing": True})
+    @patches
+    @torch.no_grad
+    @parametrize("has_bias", [True, False])
+    # Tolerance may need to be adjusted with BFloat16
+    @parametrize("dtype", [torch.float])
+    @parametrize("per_channel_quant", [True, False])
+    @parametrize("reshape_a", [True, False])
+    def test_da8w8_sym_act_sym_wgt_with_int_mm(
+        self, has_bias, dtype, per_channel_quant, reshape_a
+    ):
+        r"""
+        This testcase check if we can match the int8_dynamic_activation_int8_weight int8 linear pattern from torchao,
+        when activation is symmetrically quantized dynamically & weights are symmetrically quantized (statically)
+        The pattern is:
+            (no bias) _int_mm -> convert_element_type -> ([expand_a] -> mul) -> mul
+        or
+            (with bias) pattern_no_bias -> add
+        Expansion of the scale of activation is optional.
+        The pattern depiction doesn't mean that convert_element_type output is fed into expand_a as input,
+        but simply that activation scale may be applied after an expand operation on it.
+        """
+        if dtype == torch.bfloat16 and not torch.ops.mkldnn._is_mkldnn_bf16_supported():
```

Comment:
Do we support `dtype==torch.bfloat16` in this UT? It seems you added the `bf16` test in `test_mkldnn_pattern_matcher` but not here.